app/components/WalletConnect.tsx

```diff
@@ -468,7 +468,25 @@ export default function WalletConnect({
         throw new Error("Failed to deploy Safe");
       }
 
-      // Step 3: Complete the setup
+      // Step 3: Register account with backend
+      const accountResponse = await fetch(
```

Comment:
I'm confused by this change. Are we using the backend now? Why are we merging logic that uses `localhost:3030` into main?

Reply:
ohh yeah I missed this was merged into [feat/switch-to-sepolia](https://github.com/zkemail/email-signer-frontend/tree/feat/switch-to-sepolia)